pgrx/src/memcx.rs

```diff
@@ -16,27 +18,35 @@ pub struct MemCx<'mcx> {
 }
 
 impl<'mcx> MemCx<'mcx> {
-    /// You probably shouldn't be using this.
+    /// Wrap the provided [`pg_sys::MemoryContext`]
+    ///
+    /// # Safety:
```

Comment:
This line gets formatted as a title.
```suggestion
    /// # Safety
```

Reply:
Corrected in https://github.com/pgcentralfoundation/pgrx/pull/1737/commits/f149072c318f7e39c212daf3b59e0f723d1dfd46